### DISCUSSION TITLE
分享按钮复制的网址参数清理

### DISCUSSION THREAD

**Discussion by Temp314159** [the1812/Bilibili-Evolved#4553] [功能建议 / Ideas]
网址参数清理组件可以改变浏览器访问的网址，但点击分享按钮复制的网址仍是带参数的
虽然可以手动复制网址，但分享生成的文本会带上标题，分享时内容更直观，操作更便捷
希望有组件能删除复制内容的参数，或将功能直接并入网址参数清理中

**Comment by Geeyun-JY3** [comment]
如果只是要复制链接（和标题等信息）来分享，我会更倾向于另起炉灶来处理复制出来的链接（和 N 多自定义格式），我觉得破站自带的这个分享按钮除了在光标悬停出来的面板勾选`从 xx:xx 开始分享`时可以看到空降点画面外就没别的什么优势了，没必要死守这东西不放（还有人希望屏蔽到这个分享按钮呢：#3663，该需求也已实现：[`隐藏视频分享`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#隐藏视频分享)），之前有个脚本就可以做到自定义复制的内容格式：<https://greasyfork.org/zh-CN/scripts/398655>（可惜现在已经失效了）

当然我个人并不清楚实现你所说的网址参数清理的难度和可行性。

**Comment by the1812** [comment]
可以整合到 `BV 号转换` 的复制功能里

**Reply by the1812** [reply]
不对, 不用整合了, 本来就有
![Snipaste_2024-01-14_14-58-39](https://github.com/the1812/Bilibili-Evolved/assets/26504152/84deaf39-5262-4e73-aceb-7ec8cc83c524)
